raphtory/src/algorithms/motifs/three_node_temporal_motifs.rs

```diff
@@ -387,8 +388,10 @@ pub fn triangle_motifs<G: GraphViewOps>(
         |_, _, _els, local| {
             let mut tri_motifs = HashMap::new();
             for (vref, mc) in enumerate(local) {
-                let v_gid = graph.vertex_name(vref.into());
-                tri_motifs.insert(v_gid.clone(), mc.triangle);
+                if graph.has_vertex_ref(VertexRef::Local(vref.into())) {
```

Comment:
Is it possible to make the enumerate function not return the vref if it doesn't exist in the local graph? Maybe we need to different enumerate functions for local

Reply:
That is the Ticket @Haaroon has allocated to him